packages/liveblocks-react-ui/src/primitives/Composer/index.tsx

```diff
@@ -1198,9 +1219,13 @@ const ComposerForm = forwardRef<HTMLFormElement, ComposerFormProps>(
           onSubmitEnd();
         }
       },
-      [editor, attachments, onComposerSubmit, onSubmit, onSubmitEnd]
+      [disabled, editor, attachments, onComposerSubmit, onSubmit, onSubmitEnd]
     );
 
+    const stopPropagation = useCallback((event: SyntheticEvent) => {
+      event.stopPropagation();
+    }, []);
```

Comment:
Is this is the hidden input used for attachment upload? Was this previously causing an issue?

Reply:
Yes that's the one, we didn't get any reports but we stop propagation for all `onClick` events so we should do it here as well because it can be triggered by clicking on the "Add attachments" button.